<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Croop</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="PictureDisplay" name="label_Picture">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 222, 222);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Coordinate">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>530</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_unlabeledData">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Unlabeled Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_previousPicture">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>530</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nextPixture">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>530</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_crop">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>530</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>351</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_search">
         <property name="text">
          <string>Search:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_search"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView_photoWithDirection"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_referenceData">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reference Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_labeledData">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Labeled Data</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_photoWithDirection">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>540</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_revomeSelection">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>580</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Selection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addNewModel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>630</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_goIndex">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>590</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_goPicture">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>590</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Index:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_goIndex">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>630</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_index_currentPic">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_slash1">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_number_totalPic">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>listWidget_photoWithDirection</zorder>
   <zorder>pushButton_crop</zorder>
   <zorder>label_Picture</zorder>
   <zorder>label_Coordinate</zorder>
   <zorder>pushButton_unlabeledData</zorder>
   <zorder>pushButton_previousPicture</zorder>
   <zorder>pushButton_nextPixture</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pushButton_referenceData</zorder>
   <zorder>pushButton_labeledData</zorder>
   <zorder>pushButton_revomeSelection</zorder>
   <zorder>pushButton_addNewModel</zorder>
   <zorder>lineEdit_goIndex</zorder>
   <zorder>label_goPicture</zorder>
   <zorder>pushButton_goIndex</zorder>
   <zorder>layoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PictureDisplay</class>
   <extends>QLabel</extends>
   <header location="global">picturedisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
